src/components/text/heading/style.ts

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+import { HeadingProps } from './index'
+import { setTheme } from '../../../helpers'
+
+// All feature-based text inherits from StyledSharedHeading
+// which excludes normal h1...h6 headings
+const StyledSharedHeading = styled.span`
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+  -webkit-font-smoothing: antialiased;
+` as any
+
+const StyledH1 = StyledSharedHeading.withComponent('h1').extend`
```

Comment:
Please set headers based on the design language.

h1 48px
h2 38px
h3 28px
h4 22px
h5 16px
h6 14px
h7 12px